include/SugarObjects/templates/basic/Basic.php

```diff
@@ -160,7 +204,9 @@ private function getEmailAddressId($emailField)
         }
 
         if (!$found) {
-            throw new RuntimeException('A Basic bean has not selected email address. (' . $emailAddress . ')');
+            // Changed exception to error as demo data is never selected.
```

Comment:
If you remove the exception, perhaps you also wanted to change the phpdoc above.

Reply:
Php Doc Updated